src/docs/implementations/new-relic.adoc

```diff
@@ -6,6 +6,10 @@ Jon Schneider <jschneider@pivotal.io>
 
 New Relic offers a dimensional monitoring system product called Insights with a full UI and a query language called NRQL. New Relic Insights operates on a push model. Some features of NRQL assume that Insights receives a distinct event payload for every timing, count, etc. Micrometer instead ships aggregates at a prescribed interval, allowing your app's throughput to scale without concern for event propagation to Insights becoming a bottleneck.
 
+NOTE: New Relic provides its own Micrometer `MeterRegistry` implementation based on dimensional metrics.
+It is intended to supersede Micrometer's `NewRelicMeterRegistry` using custom events as dimensional metrics are a better fit for metrics than custom events.
```

Comment:
```suggestion
It intends to supersede Micrometer's `NewRelicMeterRegistry` (which uses custom events in New Relic) because New Relic's dimensional metrics are a better fit for metrics than custom events.
```

Reply:
@shakuzen Thanks! I applied it in 94af8a6.